modin/backends/pandas/query_compiler.py

```diff
@@ -2481,6 +2507,25 @@ def groupby_agg(
         groupby_kwargs,
         drop=False,
     ):
+        def is_reduce_fn(o, deep_level=0):
```

Comment:
It would be nice if you could add a few words about `deep_level` parameter.

Reply:
we call `is_reduce_fn` for every "value" in an aggregation dictionary with `deep_level=0` initially, on that level we can get either function name and check if it is a reduction function, or we can get a sequence of function names, if it is a sequence then we call `is_reduce_fn` for every value of that sequence with `deep_level=1`, on that level we can get either function name or renamer.

Renamer is some container with length == 2 and which first component is the new column name and the second - function to apply:
```python
agg_dict = {"col1": ["max", ("new_name", "mean")]}
```

Reply:
It's clearer at once. :) Please document this in code.

Reply:
done